autoload/neomake/makers/ft/rst.vim

```diff
@@ -23,3 +23,19 @@ function! neomake#makers#ft#rst#rstcheck()
             \ '%E%f:%l: (SEVERE/4) %m',
         \ }
 endfunction
+
+" TODO: determine proper path of current .rst file upwards in sear for conf.py file, use that instead of "docs"
+" TODO: ask noomake for a temporary folder instead of /tmp/neomake-sphinx
+" TODO: split arguments and make them configurable
```

Comment:
You should use a list instead of a string anyway.
And then it can be configured/overwritten through `g:neomake_rst_sphinx_args` already.
But you probably mean to allow for some additional configuration?!  What parts?